src/compiler/checker.ts

```diff
@@ -3762,6 +3775,12 @@ namespace ts {
             return getTypeOfGlobalSymbol(getGlobalTypeSymbol(name), arity);
         }
 
+        function getExportedTypeOfNamespace(namespace: string, name: string): Type {

```

Comment:
I misunderstood this to be "get a type defined by the exports of a namespace". I think a better name would be `getExportedTypeFromNamespace` and document what this does with a quick JSDoc comment..